src/views/settings.py

```diff
@@ -71,6 +71,7 @@ def __init__(self, parent_frame: ttk.Frame,
       self.start_button = ttk.Button(self.root, text = "Start", bootstyle=SUCCESS, command=on_start)
       self.reset_button = ttk.Button(self.root, text = "Reset", bootstyle=SECONDARY, command=self._on_reset)
       self.quit_button = ttk.Button(self.root, text = "Quit", bootstyle=DANGER, command=on_quit)
+      self.pause_button = ttk.Button(self.root, text = "Pause", bootstyle=SECONDARY, command=on_quit)
```

Comment:
We agreed on next and stop buttons. so this is not needed (and also not related to this PR)